packages/docs/getting-started/running-attestation-service.md

```diff
@@ -56,39 +57,19 @@ If you prefer using Twilio, this is list of the variables to use:
 
 ## Accounts Configuration
 
-First we need to create an account for getting the attestation key needed to sign the attestations. Run:
+First, the validator account needs to authorize the attestation signer. With `celocli` connected to a node that has the attestation signer unlocked, let's create a proof-of-posession:
```

Comment:
First, a separate key needs to be authorized with which to sign attestations. To authorize this key, we first need to create a proof-of-possession. This proof-of-possession is simply a signature, by the signing key, on the address that is authorizing the signer.

Reply:
Will move this to the improved validator setup docs PR